docs/components/Combobox/Combobox.stories.mdx

```diff
@@ -111,10 +126,6 @@ be used in the
 [empty state](../?path=/story/components-selections-combobox-web--empty-state)
 of the Combobox, as well as the noun within the search placeholder.
 
-Follow InputText content guidelines for the text input.
-
-Follow Button content guidelines for the actions.
-
```

Comment:
This seemed redundant to me, considering we are linking to the product vocabulary page just a few sentences before this.